Nautilus/Assets/Gadgets/GadgetExtensions.cs

```diff
@@ -147,6 +148,31 @@ public static EquipmentGadget SetEquipment(this ICustomPrefab customPrefab, Equi
         return equipmentGadget;
     }
 
+    /// <summary>
+    /// Sets this item as a vehicle upgrade module. Cyclops upgrades are not supported by this function.
+    /// <para>If you're using this function, please do not use <see cref="SetEquipment(ICustomPrefab, EquipmentType)"/>,<br/>
+    /// it would interefere with this and possibly make the game crash or the mod not to work.</para>
```

Comment:
Grammar nitpick: Change "the mod not to work" to "cause the mod to not work", or something similar.

However, calling SetEquipment wouldn't do either thing, I don't think, in fact it would do nothing at all since you are using the gadget extensions. Those are always safe.

The rest of these docs are good btw.

Reply:
Oh okay, I'll fix that thanks I was really not sure when I was writing :D

Reply:
Well, I must at least give an error if the item has no Equipment gadget...